### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by joeyevoli**
@spinosa I got your pull request right 'errrrr.

### CODE REVIEW

File: sample_app/config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  
+  root 'static_pages#home'
+
+  get 'static_pages/home'
+  get 'static_pages/help'
+  get 'static_pages/about'
+  get 'static_pages/contact'
```
Comment (spinosa) [inline sample_app/config/routes.rb:8]: This line works.  But it makes the contact page available at "http://host/static_pages/contact".  Is that what you want?  Or do you want the page to be at "http://host/contact" ?

(similar for all the other ones above)

Comment (joeyevoli) [inline sample_app/config/routes.rb:8]: I'm pretty positive the tutorial walks me through how to change that in later chapters.  Let's leave it for now.

**Comment by spinosa** [comment]
@joeyevoli in that case, LGTM!
(customary for the branch to owner to do the merge, so i'll leave it for you)

**Comment by joeyevoli** [comment]
@spinosa All good?

**Comment by spinosa** [comment]
Looks merged. Next time you can merge it from the CLI instead of GH. See
that it's not magic

**Pull request merged**